src/Components/FormCreation/CreateEditForm.js

```diff
@@ -125,36 +119,29 @@ function CreateEditForm() {
   }
 
   return (
-    <CreateEditFormStateContext.Provider value={formState}>
-      <CreateEditFormDispatchContext.Provider value={dispatch}>
-        <Wrapper>
-          <CreateEditFormHeader />
-          {form.sections &&
-            form.sections.map((section, key) => (
-              <FormWrapper key={key}>
-                <FormSection
-                  key={key + "_section"}
-                  numSections={form.sections.length}
-                  section={key + 1}
-                  title={section.title}
-                  description={section.desc}
-                />
-                {section.questions.map((_question, questionKey) => (
-                  <FormCard
-                    key={questionKey + "_question"}
-                    active={
-                      activeSection === key && activeQuestion === questionKey
-                    }
-                    handleActive={updateActive}
-                    sectionKey={key}
-                    questionKey={questionKey}
-                  />
-                ))}
-              </FormWrapper>
+    <Wrapper>
+      <CreateEditFormHeader {...headerData} onChange={setHeaderData} />
+      {form.sections &&
+        form.sections.map((section, key) => (
+          <FormWrapper key={key}>
+            <FormSection
+              key={key + "_section"}
+              numSections={form.sections.length}
+              sectionNum={key + 1}
+              sectionData={section}
+            />
+            {section.questions.map((_question, questionKey) => (
+              <FormCard
+                key={questionKey + "_question"}
+                active={activeSection === key && activeQuestion === questionKey}
```

Comment:
Once we render the questions inside FormSection, we can move the active question logic to form section as well and set the active question by whether or not the current section is active.